src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from 'react-router-dom';
+
+import Details from './components/Details/index';
+import Countries from './components/Countries/index';
+
+import './App.css';
+import Header from './components/Header';
+
+const App = () => (
+  <>
+    <Header />
+    <Routes>
+      <Route exact path="/" element={<Countries />} />
+      <Route path="/details" element={<Details />} />
```

Comment:
- [ ] According to the requirements, each category item detail page should have its unique route within the SPA. I'd suggest learning how to use URL parameters that react router provides.